src/types/controller.ts

```diff
@@ -78,18 +78,30 @@ export type UseControllerReturn<
  * />
  * ```
  */
+
+type ControllerRender<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>,
+> = ({
+  field,
+  fieldState,
+  formState,
+}: {
+  field: ControllerRenderProps<TFieldValues, TName>;
+  fieldState: ControllerFieldState;
+  formState: UseFormStateReturn<TFieldValues>;
+}) => React.ReactElement;
+
 export type ControllerProps<
   TFieldValues extends FieldValues = FieldValues,
   TName extends FieldPath<TFieldValues> = FieldPath<TFieldValues>,
   TTransformedValues = TFieldValues,
-> = {
-  render: ({
-    field,
-    fieldState,
-    formState,
-  }: {
-    field: ControllerRenderProps<TFieldValues, TName>;
-    fieldState: ControllerFieldState;
-    formState: UseFormStateReturn<TFieldValues>;
-  }) => React.ReactElement;
-} & UseControllerProps<TFieldValues, TName, TTransformedValues>;
+> = (
+  | {
+      render: ControllerRender<TFieldValues, TName>;
+    }
+  | {
+      children: ControllerRender<TFieldValues, TName>;
+    }
+) &
+  UseControllerProps<TFieldValues, TName, TTransformedValues>;
```

Comment:
Currently, how does `children` work for the controller? Basically nothing happens?